inc/class-core-sitemaps-renderer.php

```diff
@@ -91,6 +91,22 @@ public function get_sitemap_index_stylesheet_url() {
 	public function render_index( $sitemaps ) {
 		header( 'Content-type: application/xml; charset=UTF-8' );
 
+		if ( ! class_exists( 'SimpleXMLElement' ) ) {
+			add_filter( 'wp_die_handler', static function() { return '_xml_wp_die_handler'; } );
```

Comment:
this line seems to throw phpcs errors

Reply:
Ah yeah, need to add some newlines probably.